ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONClient.java

```diff
@@ -4,9 +4,16 @@
 import eu.chargetime.ocpp.feature.profile.Profile;
 import eu.chargetime.ocpp.model.Confirmation;
 import eu.chargetime.ocpp.model.Request;
+import eu.chargetime.ocpp.wss.BaseWssSocketBuilder;
+import eu.chargetime.ocpp.wss.WssSocketBuilder;
+import org.java_websocket.drafts.Draft;
+import org.java_websocket.drafts.Draft_6455;
+import org.java_websocket.protocols.Protocol;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import javax.net.ssl.SSLContext;
```

Comment:
Reverting the external API changes makes this line necessary again. A merge problem. :)

Reply:
That's true. Fixed.